core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -1111,30 +1116,28 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       sealRoute(services.adminRoutes) ~>
       check {
         assertResult(StatusCodes.OK) { status }
-        responseAs[Array[Workspace]] should contain
-        theSameElementsAs(Array(testData.workspace, testData.workspaceNoGroups, testData.workspacePublished, testData.workspaceNoAttrs))
+        // TODO: why is this result returned out of order?
+        sortAndAssertWorkspaceResult(testData.allWorkspaces) { responseAs[Seq[Workspace]] }
       }
   }
 
-  it should "return 200 when getting workspaces by a string attribute" in withTestDataApiServices { services =>
+  it should "return 200 when getting workspaces by a string attribute" in withConstantTestDataApiServices { services =>
```

Comment:
results are less interesting here and below for the Constant Test Data - I can switch back if we prefer that